### ISSUE TITLE
Boost.Graph no longer builds with C++03

### ISSUE THREAD

**Issue by mclow** [boostorg/graph#353]
It includes `lexical_cast.h` which no longer supports C++03.

**Comment by mclow** [comment]
Discovered in Boost 1.84.0 Beta 1 Release Candidate 1

**Comment by jeremy-murphy** [comment]
Is that actually an issue though? I thought all libraries were free to dispense with supporting pre-C++11 now? I've already taken the C+98 and C++03 CI builders out of the configurations for Drone and GitHub actions.

**Comment by mclow** [comment]
If you decide that Boost.Graph is no longer supporting c++03, that's your call.   

But when I look at the docs and the meta/library.json file, I don't see any suggestion of that.

Many libraries have (or are in the process of) dispensing with C++03 support.
I see messages like this in my build logs:

./boost/smart_ptr/detail/requires_cxx11.hpp:19:1: warning: C++03 support was deprecated in Boost.SmartPtr 1.82 and will be removed in Boost.SmartPtr 1.84. Please open an issue in https://github.com/boostorg/smart_ptr if you want it retained. [-W#pragma-messages]

**Comment by jeremy-murphy** [comment]
OK, good point, I should communicate it more explicitly.

**Comment by andrea-cassioli-maersk** [comment]
What has been the decision here? I see still 03 in the meta/library.json file.

**Comment by jeremy-murphy** [comment]
The decision is to drop C++03, the fact is I haven't had time to figure out how to do it.

**Comment by andrea-cassioli-maersk** [comment]
I guess this issue is settled now. Shall it be close?

**Comment by jeremy-murphy** [comment]
Yes, I'm pretty sure I actually fixed this by updating the metadata.
